Packs/GreatHorn/Integration/GreatHorn/GreatHorn.yml

```diff
@@ -0,0 +1,868 @@
+commonfields:
+  id: GreatHorn
+  version: -1
+fromversion: 5.0.0
+name: GreatHorn
+display: GreatHorn
+category: Email Gateway
+description: The only cloud-native security platform that stops targeted social engineering and phishing attacks on cloud email platforms like Office 365 and G Suite.
+configuration:
+- display: Base URL
+  name: url
+  type: 0
+  required: true
+- display: API Version
+  name: api_version
+  defaultvalue: v2
+  type: 0
+  required: true
+- display: API Key
+  name: apikey
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Trust any certificate (not secure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: ''
+  type: python
+  commands:
+  - name: gh-get-message
+    description: Return message details for the specified event
+    arguments:
+    - name: id
+      required: true
+      description: GreatHorn eventId, multiple values supported via CSV
```

Comment:
Please note: If a command uses argToList function the argument `isArray` should be set to true.

Reply:
done